region/maridia/inner-pink.json

```diff
@@ -2613,13 +1764,11 @@
             {
               "id": 1,
               "requires": [
-                {
-                  "or": [
-                    "HiJump",
-                    "SpaceJump",
-                    "canTrickyWallJump"
-                  ]
-                },
+                {"or": [
+                  "HiJump",
```

Comment:
I think I'd like to see a canWalljump even with HiJump because of the last two jumps into the door.